libs/azure-api-client/src/Authentication/AuthenticatorInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\AzureApiClient\Authentication;
+
+use Keboola\AzureApiClient\GuzzleClientFactory;
+
+interface AuthenticatorInterface
+{
+    public function __construct(GuzzleClientFactory $clientFactory);
+
+    public function getAuthenticationToken(string $resource): string;
+
+    public function checkUsability(): void;
```

Comment:
nemyslím si, že by to mělo být součástí toho interfacu

Reply:
dtto https://github.com/keboola/azure-key-vault-php-client/blob/master/src/Authentication/AuthenticatorInterface.php#L15